js/graph_spectrum_plot.js

```diff
@@ -237,101 +237,121 @@ GraphSpectrumPlot._drawHeatMap = function() {
         }
     }
 
-    // The resulting image has imperfections, usually we not have all the data in the input, so we apply a little of blur 
+    // The resulting image has imperfections, usually we not have all the data in the input, so we apply a little of blur
     canvasCtx.filter = `blur(${BLUR_FILTER_PIXEL}px)`;
     canvasCtx.drawImage(heatMapCanvas, 0, 0);
     canvasCtx.filter = 'none';
 
     return heatMapCanvas;
-}
+};
 
 GraphSpectrumPlot._drawFiltersAndMarkers = function(canvasCtx) {
 
-    var HEIGHT = this._canvasCtx.canvas.height - MARGIN;
-    var WIDTH  = this._canvasCtx.canvas.width - this._getActualMarginLeft();
-    var PLOTTED_BLACKBOX_RATE = this._fftData.blackBoxRate / this._zoomX;
+    const HEIGHT = this._canvasCtx.canvas.height - MARGIN;
+    const WIDTH  = this._canvasCtx.canvas.width - this._getActualMarginLeft();
+    const PLOTTED_BLACKBOX_RATE = this._fftData.blackBoxRate / this._zoomX;
 
-    var offset = 2; // make some space! Includes the space for the mouse frequency. In this way the other elements don't move in the screen when used
+    let offset = 2; // make some space! Includes the space for the mouse frequency. In this way the other elements don't move in the screen when used
 
     // Gyro filters
-    if (this._overdrawType == SPECTRUM_OVERDRAW_TYPE.ALL_FILTERS ||
-        this._overdrawType == SPECTRUM_OVERDRAW_TYPE.GYRO_FILTERS ||
-       (this._overdrawType == SPECTRUM_OVERDRAW_TYPE.AUTO && this._fftData.fieldName.match(/(.*gyro.*)/i) != null)) {
+    if (this._overdrawType === SPECTRUM_OVERDRAW_TYPE.ALL_FILTERS ||
+        this._overdrawType === SPECTRUM_OVERDRAW_TYPE.GYRO_FILTERS ||
+       (this._overdrawType === SPECTRUM_OVERDRAW_TYPE.AUTO && this._fftData.fieldName.toLowerCase().indexOf('gyro') !== -1)) {
 
-        // Dynamic gyro lpf 
+        // Dynamic gyro lpf
         if(this._sysConfig.gyro_lowpass_dyn_hz[0] != null && this._sysConfig.gyro_lowpass_dyn_hz[0] > 0 &&
                 this._sysConfig.gyro_lowpass_dyn_hz[1] > this._sysConfig.gyro_lowpass_dyn_hz[0]) {
-            let label = this._sysConfig.gyro_soft_type != null ? `GYRO LPF (${FILTER_TYPE[this._sysConfig.gyro_soft_type]}) Dyn cutoff` : 'GYRO LPF Dyn cutoff';
-            this._drawLowpassDynFilter(canvasCtx, this._sysConfig.gyro_lowpass_dyn_hz[0], this._sysConfig.gyro_lowpass_dyn_hz[1], PLOTTED_BLACKBOX_RATE, label, WIDTH, HEIGHT, (15*offset++) + MARGIN, "rgba(94, 194, 98, 0.50)");
+            const label = this._sysConfig.gyro_soft_type != null ? `GYRO LPF (${FILTER_TYPE[this._sysConfig.gyro_soft_type]}) Dyn cutoff` : 'GYRO LPF Dyn cutoff';
+            this._drawLowpassDynFilter(canvasCtx, this._sysConfig.gyro_lowpass_dyn_hz[0], this._sysConfig.gyro_lowpass_dyn_hz[1], PLOTTED_BLACKBOX_RATE, label,
+                                       WIDTH, HEIGHT, (15 * offset) + MARGIN, "rgba(94, 194, 98, 0.50)");
```

Comment:
Not sure we want to go for breaking these lines up - today's editors are actually quite good at handling long lines.
And even more so, we should not go for 'fancy' text alignment - if a line is broken, the follow-on line should be exactly one tab width indented - fancy alignment just results in harder to maintain code (i.e. if the function is renamed at some point, then all the places where it is called will have to have the indentation manually fixed)

Reply:
Here I broke more lines than requested from Sonar. Only two of them were requested. 
We have a rule to not let more than 180 chars length. What we want to do? Let some limit? Is 180 a good size? Some bigger size? Or remove completely the rule?

Reply:
I think the 180 character limit is fine - I don't think that breaking up lines or not has to be a hard and fast rule.
On the other hand I see adding additional whitespace to align lines as problematic, as this introduces unneeded dependencies between lines which are prone to require changes in otherwise unchanged lines down the track, making maintenance and review of code harder for not much benefit.

Reply:
I don't understand what are you exactly saying about adding additional whitespace... Do you say that if I break a line the second one must go more to the left? Maybe only four spaces more than the first for example?

Reply:
If a line is broken, the second and all following lines should always be indented 4 spaces more than the first.